### ISSUE TITLE
the wrong 2d plot 

### ISSUE THREAD

**Issue by zchky** [lyakaap/image-feature-learning-pytorch#2]
if i directly run the code. 

i find the 2d plot on Contrastive center loss is similar to softmax's 2d deeply feature.

Could you help me on that?
